src/main/kotlin/org/rust/ide/presentation/Utils.kt

```diff
@@ -57,7 +57,11 @@ fun getPresentationForStructure(psi: RsElement): ItemPresentation {
         }
     }
     val icon = psi.getIcon(Iconable.ICON_FLAG_VISIBILITY)
-    val textAttributes = if (psi.isExpandedFromMacro) RsColor.GENERATED_ITEM.textAttributesKey else null
+    val textAttributes = when {
+        psi.isExpandedFromMacro -> RsColor.GENERATED_ITEM.textAttributesKey
+        psi is RsDocAndAttributeOwner && !psi.isEnabledByCfgSelf -> RsColor.CFG_DISABLED_CODE.textAttributesKey
```

Comment:
Maybe it's better to swap these lines to show `RsColor.CFG_DISABLED_CODE.textAttributesKey` for all disabled items even they are generated? I suppose it's ore important for users

Reply:
Ok, sounds reasonable